Explore teaching advisers

mariosh-e opened the pull request:
**Trello card**
https://trello.com/c/tNGVLNX6

**Context**
As we are improving the design of our subject pages, we need to update the design of the Explore teaching advisers page so it is more consistent with our other pages

**Changes proposed in this pull request**
Revise how the bursary page is organised

emmaachesongray commented on app/views/content/explore-teaching-advisers.md:
```diff
@@ -1,5 +1,8 @@
 title: Explore teaching advisers
+title_paragraph: |-
+  <p>
+  Finding out which career is right for you is not always easy. How do you know what something is like if you've never done it before?</p>
```
```suggestion
  Our explore teaching advisers can help you discover whether the classroom is where you belong. Your dedicated adviser will give you free support and guidance, and can answer all your questions about what teaching is really like.</p>
```

Suggest removing this line, and using the first 2 sentences from the other paragraph as more effective.

emmaachesongray commented on app/views/content/explore-teaching-advisers/_article.html.erb:
```diff
@@ -1,56 +1,11 @@
-    </div>
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Find your future career</h2>
+    <p>Our explore teaching advisers can help you discover whether the classroom is where you belong. Your dedicated adviser will give you free support and guidance, and can answer all your questions about what teaching is really like.</p>
```
Remove first 2 sentences as they're moved above.

emmaachesongray commented on app/views/content/explore-teaching-advisers/_ending.html.erb:
```diff
@@ -1,44 +1,24 @@
+  <section class="col col-720 col-space-l-top">
+  <h2 class="heading--box-blue">Is teaching right for me?</h2>
+  <h3>Use your subject passion</h3>
+  <p>Bring your subject to life for your pupils. Help them be as inspired by it as you are.</p>
```
Not sure of value to users of this section, especially if main page aim is to get people to sign up to the advisers? Let's discuss tomorrow.

emmaachesongray reviewed (approved):
LGTM 👍

martyn-w commented:
> @martyn-w @jenhadfield-dfe The **_ending.html.erb_** content is not needed anymore but I didn't want to delete the file as it says its permanent. I don't think the file is anywhere else but wanted to check the best method to do this - delete content or the whole file. I wont merge until this is confirmed. Thank you

Hi @mariosh-e , you can safely delete the file `app/views/content/explore-teaching-advisers/_ending.html.erb` but you must also remove the reference to it: i.e. delete line 17 of `app/views/content/explore-teaching-advisers.md`

mariosh-e commented:
@martyn-w @jenhadfield-dfe I have deleted the ending file and deleted the reference on explore-teaching-advisers.md too. Ready for final review then I can merge. Thank you

martyn-w reviewed (approved):
👍 LGTM from a technical perspective - please also get a reviewer for the content

emmaachesongray reviewed (approved):
All good from content POV 🥇
